src/pages/home/Home.page.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { RiHandHeartFill } from "react-icons/ri";
+import { FaShippingFast } from "react-icons/fa";
+import CategoryCard from "../../components/categoryCard/CategoryCard.component";
+import ProductCard from "../../components/productCard/ProductCard.component";
+import feauturedProducts from "../../utils/mocks/featured-products.json";
+import feauturedCategories from "../../utils/mocks/product-categories.json";
+import {
+  StyledDescription,
+  StyledIcon,
+  StyledTitle,
+  StyledTitleInformaton,
+  StyledWrapper,
+  StyledWrapperCategories,
+  StyledWrapperDescription,
+  StyledWrapperInformation,
+  StyledWrapperTitle,
+} from "./Home.styles";
+import Slider from "../../components/slider/Slider.component";
+
+// eslint-disable-next-line react/prop-types
+function HomePage() {
+  const products = feauturedProducts.results;
```

Comment:
Let's consider that we cannot be certain if we will be successful obtaining the information. That is why we can implement showing a message in case of _error_ and _isLoading_ while we are retrieving the information, also having an empty array [] as the default value will help prevent the component to stop the whole page to render. : 

```
const products = !feauturedProducts.results.length
    ? feauturedProducts.results
    : [];
```
Not really necessary to apply since we would add the API request later, but still worth mentioning